docs/Learning-Environment-Design-Agents.md

```diff
@@ -175,18 +174,38 @@ For angles that can be outside the range [0,360], you can either reduce the
 angle, or, if the number of turns is significant, increase the maximum value
 used in your normalization formula.
 
-### Multiple Visual Observations
-
-Visual observations use rendered textures directly or from one or more
-cameras in a scene. The Policy vectorizes the textures into a 3D Tensor which
-can be fed into a convolutional neural network (CNN). For more information on
-CNNs, see [this guide](http://cs231n.github.io/convolutional-networks/). You
-can use visual observations along side vector observations.
+#### Vector Observation Summary & Best Practices
+
+* Vector Observations should include all variables relevant to allowing the
+  agent to take the optimally informed decision, and ideally no extraneous information.
+* In cases where Vector Observations need to be remembered or compared over
+  time, either an LSTM (see [here](Feature-Memory.md)) or by changing the
+  `Stacked Vectors` value in the agent GameObject's `Behavior Parameters`.
+* Categorical variables such as type of object (Sword, Shield, Bow) should be
+  encoded in one-hot fashion (i.e. `3` -> `0, 0, 1`).
+* In general, all inputs should be normalized to be in
+  the range 0 to +1 (or -1 to 1). For example, the `x` position information of
```

Comment:
This information is not present in the above text (I think). Maybe this section should just be "Best practices".
It feels weird to have a section that is both a summary and a best practice...

Reply:
I am open to different ideas for how to organize the information or name these sections.